devtools/packaging/dlc-games.json

```diff
@@ -0,0 +1,62 @@
+{
+	"beneath-a-steel-sky": {
+		"name": "Beneath a Steel Sky - Freeware CD Version",
+		"packname": "beneath_a_steel_sky",
+		"description": "A science-fiction thriller set in a bleak post-apocalyptic vision of the future, Beneath a Steel Sky revolves around \"Union City\", where selfishness, rivalry, and corruption by its citizens seems to be all too common, those who can afford it live underground, away from the pollution and social problems which are plaguing the city. \n\nYou take on the role of Robert Foster, an outcast of sorts from the city since a boy who was raised in a remote environment outside of Union City simply termed \"the gap\". Robert's mother took him away from Union City as a child on their way to \"Hobart\" but the helicopter crashed on its way, unfortunately Robert's mother dies, but he survives and is left to be raised by a local tribe from the gap. \n\nYears later, Union City security drops by and abducts Robert, killing his tribe in the process; upon reaching the city the helicopter taking him there crashes with him escaping, high upon a tower block in the middle of the city he sets out to discover the truth about his past, and to seek vengeance for the killing of his tribe. \n\nNote that this package only contains game-data. The game engine is provided by ScummVM.",
+		"shortDescription": "Adventure sci-fi thriller game for ScummVM",
+		"supportedLanguages": {
+			"interface": ["english"],
+			"audio": ["english"],
+			"subtitles": ["english"]
+		},
+		"genres": ["Adventure", "Free to Play", "Point & Click", "Cyberpunk"],
+		"storeCategories": ["Games"],
+		"supportContactInfo": {
+			"websiteURL": "https://www.scummvm.org/"
+		},
+		"graphicalAssets": {
+			"screenshots": [
+				"https://www.scummvm.org/data/screenshots/sky/sky/sky_dos_en_1_1.jpg",
+				"https://www.scummvm.org/data/screenshots/sky/sky/sky_dos_en_1_2.jpg"
+			],
+			"StoreLogo": "https://raw.githubusercontent.com/scummvm/scummvm-icons/master/icons/sky-sky.png"
+		},
+		"scummVMConfigData": {
+			"description": "Beneath a Steel Sky (v0.0372 CD/DOS)",
+			"engineid": "sky",
+			"extra": "v0.0372 CD",
+			"gameid": "sky",
+			"guioptions": "lang_English (GB) lang_German lang_French lang_English (US) lang_Swedish lang_Italian lang_Portuguese (Brazil) lang_Spanish",
+			"language": "",
+			"platform": "pc"
+		},
+		"size": "69377781",
+		"url": "https://downloads.scummvm.org/frs/extras/Beneath%20a%20Steel%20Sky/bass-cd-1.2.zip"
```

Comment:
How to describe a game that consists of many files?

Reply:
@sev- Can you give an example? Do you mean the game is available to download in multiple .zip parts (due to its large size)?

Reply:
No, I mean, a game that will be represented in different packages, but that consists of many files. Or do you mean that for preparing a game for distribution, we must first zip it?